### PR TITLE
Strata - UI: Strata doc upload

### PR DESCRIPTION
**Pull request by deetz99**
*Issue:*

- bcgov/entity/issues/25006

*Description of changes:*
- 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the BC Registry and Digital Services BSD 3-Clause License

### CODE REVIEW

File: strr-host-pm-web/app/stores/document.ts
Comment (deetz99) [inline strr-host-pm-web/app/stores/document.ts:1]: Had to fix this as the details dashboard uses the same stores as the application, so if you were to view an application and then start a new one right after, the application page reset on mount would delete the documents from the previous application you were viewing

File: strr-strata-web/app/components/document/list/Item.vue
Comment (deetz99) [inline strr-strata-web/app/components/document/list/Item.vue:1]: Should probably move the document components to the base layer but that is an after christmas thing

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-419-mjp7w3n0.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-hosts-dev--pr-419-icyo7zlt.web.app

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**